kola/tests/misc/files.go

```diff
@@ -208,3 +212,148 @@ func Blacklist(c cluster.TestCluster) {
 		c.Fatalf("Blacklisted files or directories found:\n%s", output)
 	}
 }
+
+func getAllFiles(c cluster.TestCluster, m platform.Machine) map[string]string {
+	out := string(c.MustSSH(m, "sudo find / -ignore_readdir_race -mount -path /var -prune -o -type f -print0 | xargs -0 sudo md5sum"))
+	ret := map[string]string{}
```

Comment:
I think this should also get the permissions and extended attributes for comparison, as those will effect system behavior.  In the very least compare `ls -lZ`.